addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/type/HomematicTypeGeneratorImpl.java

```diff
@@ -298,7 +303,7 @@ private void generateConfigDescription(HmDevice device, URI configDescriptionURI
         for (HmChannel channel : device.getChannels()) {
             String groupName = "HMG_" + channel.getNumber();
             String groupLabel = MetadataUtils.getDescription("CHANNEL_NAME") + " " + channel.getNumber();
-            groups.add(new ConfigDescriptionParameterGroup(groupName, null, false, groupLabel, null));
+            groups.add(new ConfigDescriptionParameterGroup(groupName, null, false, groupLabel, ""));
```

Comment:
This change actually looks doggy - upon closer analysis, I created https://github.com/eclipse/smarthome/pull/4952, so that you should be able to revert this change again.